src/pycsou/runtime/__init__.py

```diff
@@ -138,13 +139,21 @@ def coerce(x):
         Fails if operation is impossible or unsafe. (I.e. casting complex-valued data.)
     """
     dtype = getPrecision().value
+    try:
```

Comment:
I don't quite agree with all the steps here. Let's talk about this in the afternoon.